### DISCUSSION TITLE
Break character to include certain strings?

### DISCUSSION THREAD

**Discussion by OceanicPianoMan** [nathanhoad/godot_dialogue_manager#209] [Q&A]
Sorry if this is posted somewhere obvious, but I am writing dialogue and I am trying to use the word "if" but it detects it as a statement rather than the intended string. 

![Screenshot 2023-07-25 032809](https://github.com/nathanhoad/godot_dialogue_manager/assets/71693321/3ee123f3-5d8a-4200-bdfc-ca82b6d88a06)
 
How would I allow for the dialogue box to display this?

**Comment by nathanhoad** [comment]
The syntax highlighting in the CodeEdit component is a little overzealous there. The compiler won't see that as a conditional even though the highlighter does.
